modules/__tests__/LocationUtils-test.js

```diff
@@ -111,4 +111,123 @@ describe('createLocation', () => {
       })
     })
   })
+
+  describe('with a current location', () => {
+    const currentLocation = {
+      pathname: '/The/Old/Path',
+      search: '?theQuery',
+      hash: '#fragment'
+    };
+
+    const createRelative = url => createLocation(url, undefined, undefined, currentLocation)
```

Comment:
Ok, should currentLocation be inlined too, or should I leave it as a `const` to avoid repeating it in every test?